source/lambda/helper/python/helper.py

```diff
@@ -123,12 +123,13 @@ def getFileNames(bucketName, prefix, maxPages, allowedFileTypes, awsRegion=None)
             else:
                 hasMoreContent = False
 
-            for doc in listObjectsResponse['Contents']:
-                docName = doc['Key']
-                docExt = FileHelper.getFileExtension(docName)
-                docExtLower = docExt.lower()
-                if(docExtLower in allowedFileTypes):
-                    files.append(docName)
+            if 'Contents' in listObjectsResponse:
+                for doc in listObjectsResponse['Contents']:
+                    docName = doc['Key']
+                    docExt = FileHelper.getFileExtension(docName)
+                    docExtLower = docExt.lower()
+                    if(docExtLower in allowedFileTypes):
+                        files.append(docName)
```

Comment:
Can we add in logs for what files were included and excluded?

Reply:
this is a general purpose listing function used in various places in DUS. Files sets in folders could be quite large and logging will likely slow down calls.